$NetBSD: patch-ac,v 1.3 2025/05/27 19:15:12 vins Exp $

* Support NetBSD curses.
* Do not look for ncursesw.
* Add DragonFly.

--- mt.h.orig	2024-08-13 10:49:48.000000000 +0000
+++ mt.h
@@ -60,12 +60,12 @@ typedef enum { SCHEME_TYPE_EDIT = 0, SCH
 #endif
 
 #if defined(UTF8_SUPPORT) && !defined(__APPLE__)
-        #if defined(__FreeBSD__) || defined (__linux__)
+        #if defined(__FreeBSD__) || defined (__linux__) || defined(__NetBSD__)
                 #include <panel.h>
                 #include <curses.h>
         #else
-                #include <ncursesw/panel.h>
-                #include <ncursesw/ncurses.h>
+                #include <ncurses/panel.h>
+                #include <ncurses/ncurses.h>
         #endif
 #else
 	#if defined(__APPLE__)
@@ -102,7 +102,7 @@ typedef enum { SCHEME_TYPE_EDIT = 0, SCH
 	#define getmaxyx(w,y,x) y = w->_maxy;  x = w->_maxx
 #endif
 
-#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__APPLE__) || defined(__CYGWIN__) || defined(__minix)
+#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__APPLE__) || defined(__CYGWIN__) || defined(__minix) || defined(__DragonFly__)
 #define off64_t off_t
 #define stat64 stat
 #define open64 open
